platformio.ini

```diff
@@ -37,5 +37,5 @@ platform = espressif8266
 board = esp12e
 framework = arduino
 lib_deps = ${common.lib_deps}
-src_build_flags = ${common.version} -DENABLE_OTA -DWIFI_LED=0 -DENABLE_DEBUG
-upload_port = openevse.local
+src_build_flags = ${common.version} -DENABLE_OTA -DWIFI_LED=0 -DENABLE_DEBUG -DHOSTNAME=openevse-ota
```

Comment:
I looked at the script option, but that depends heavily on the OS (at least, Windows can't run a bash script) ...